turbopy/core.py

```diff
@@ -170,8 +170,9 @@ def prepare_simulation(self):
         Prepares the simulation by reading the input and initializing
         physics modules and diagnostics.
         """
-        print("Reading Grid...")
-        self.read_grid_from_input()
+        if 'Grid' in self.input_data:
+            print("Reading Grid...")
+            self.read_grid_from_input()
         
```

Comment:
What do you think about adding an `else` statement here, and raising a warning that a gridless simulation is being created? This might help developers who just forget to include the grid info in the input parameters.

Reply:
Should I just print out the warning?

Reply:
You should use the [Python warnings library](https://docs.python.org/3/library/warnings.html). Something like `warnings.warn("message goes here")`.